Update/wata_001.txt

```diff
@@ -4438,7 +4439,7 @@ void main()
 	ModPlayVoiceLS(4, 24, "ps3/s19/24/992400013", 128, TRUE);
 	ModPlayVoiceLS(5, 25, "ps3/s19/25/992500010", 128, TRUE);
 	OutputLine(NULL, "「「そそ、そんなこと……そんな、あのあのあの…ッ！！！！」」",
-		   NULL, "\"\"Th-th-that... That... Uhhhhhhhhhh......!!!!!\"\"", GetGlobalFlag(GLinemodeSp));
```

Comment:
It wasn't used in Oni, and it looks quite ugly for the formatting. Thing is, if we keep it, should we change it in Oni? Also, is it consistent in future chapters?

Reply:
It is indeed consistent in future chapters.

Reply:
As for Oni we might want to fix it at some point but it's not worth to go through whole arc because of it. So I think we should just revert it here and make an issue for Oni to fix it later. What do you think?

Reply:
Actually to be more precise it is used in wata, tata and hima. Meakashi doesn't seem to use it.

Reply:
Truth be told, it's also because it was messing with my brain process when reading, because I have to set an unconscious task of noting opening and closing of dialogue with the quotation marks, to find out when one's missing or one is too much. So when there's 3 following up, it breaksit, hahaha. But I can deal with it, so it's ok, I'll revert 👌 .